test/helpers/sandbox/__main__.py

```diff
@@ -0,0 +1,103 @@
+import logging
+import pandas as pd
+from app.main import Rocket
+
+from test.helpers.dummy_device_factory import DummyGPS, DummyBrakes, DummyParachute
+from gyro import SandboxGyro
+from accelerometer import SandboxAccelerometer
+from altimeter import SandboxAltimeter
+
+
+class SandboxDeviceFactory(object):
+    def __init__(self, simulation_data):
+
+        self.accelerometer = SandboxAccelerometer(self.parse_dataframe(simulation_data, 'acceleration'))
+        self.gyro = SandboxGyro(self.parse_dataframe(simulation_data,'rate (°/s)'))
+        self.altimeter = SandboxAltimeter(self.parse_dataframe(simulation_data,'Altitude'))
+        self.gps = DummyGPS()
+        self.parachute = DummyParachute()
+
+
+    def parse_dataframe(self,dataframe,data):
+        return pd.concat([dataframe.filter(items=['# Time (s)']), dataframe.filter(like=data)],axis=1)
+
+
+class Sandbox(object):
+    def __init__(self, source_file='test/helpers/sandbox/simulation.csv'):
+        self.simulation_data = pd.read_csv(source_file)
+        self.device_factory = SandboxDeviceFactory(self.simulation_data)
+        self.rocket = Rocket(self.device_factory)
+        self.running = False
+        self.launch_time = None
+
+
+    def start(self):
+        print('Starting sandbox')
+        self.running = True
+        rocket.start()
+
+
+    def ignition(self):
+        print('Engine ignition')
+        self.launch_time = time()
+        self.device_factory.accelerometer.launch(launch_time)
+        self.device_factory.gyro.launch(launch_time)
+        self.device_factory.altimeter.launch(launch_time)
+
+
+    def stop(self):
+        print('Stopping sandbox')
+        self.running = False
+        rocket.stop()
+
+
+    def reset(self):
+        if self.running:
+            print('!!! Must stop sandbox before reseting !!!')
```

Comment:
can we use `logging.warn('Sandbox still running')`...I want to use loggers over prints as much as possible since we can easily save logs to a file with timestamps which makes remove debugging easier